### PR TITLE
[TIMOB-24495] Android: Fix textfield/textarea editable:false is not w…

### PR DESCRIPTION
**Pull request by maggieaxway**
…orking on emulator

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24495

Expect result:

-Before PR: able to input from PC keyboard to **Android emulator** TextField
-After PR: unable to input

Test case:

app.js

````js
var win = Ti.UI.createWindow(
{ backgroundColor : 'white' }
);
var textField = Ti.UI.createTextField(
{ borderStyle : Ti.UI.INPUT_BORDERSTYLE_ROUNDED, color : '#336699', top : 10, left : 10, width : 250, height : 60, value: 'edit me', editable: false }
);
win.add(textField);
win.open();

### CODE REVIEW

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java [hunk 1]
```diff
@@ -507,13 +507,15 @@ public void handleKeyboard(KrollDict d)
 
 		if (d.containsKey(TiC.PROPERTY_EDITABLE)) {
 		    editable = TiConvert.toBoolean(d, TiC.PROPERTY_EDITABLE, true);
+		    disableChangeEvent = !editable;
```
Comment (garymathews) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java]: I'm not sure this is needed?

**Review by garymathews — COMMENTED** [review]
CR: Respond to comment
FT: **PASS**

###### TEST CASE
```JS
var win = Ti.UI.createWindow({backgroundColor : 'gray', layout: 'vertical'}),
    tf = Ti.UI.createTextField({
        value: 'TEXT',
        editable: false
    }),
    btn = Ti.UI.createButton({title: 'TOGGLE EDITABLE'});

tf.addEventListener('change', function() {
    Ti.API.info('TEXT CHANGED');
});

btn.addEventListener('click', function() {
    tf.editable = !tf.editable;
});

win.add(tf);
win.add(btn);

win.open();
```

**Review by garymathews — APPROVED** [review]
CR: **PASS**
FT: **PASS**

**Review by ssjsamir — APPROVED** [review]
FR Passed - using the test case provided above by @garymathews I was not able to edit the text until the `editable` button (toggles `editable` to `true` or `false` for the text field) was pressed on android emulator.

**Test Steps**
+ Built the SDK using this PR 
+ Copied the test case in this PR and pasted it into a titanium application 
+ Ran the titanium application from Appc studio 
+ When the application launched I tried to edit the textfield but I was unable to 
+ Pressed the `toggle editable` button 
+ Now when I tried to edit the textfield I was able to edit it successfully

**Environment**
```
Appcelerator Command-Line Interface, version 6.2.0
Android emulator (Nexus 6 running android 7.0) 
Operating System Name: Mac OS X El Capitan
Operating System Version: 10.11.6
Node.js Version: 4.6.0
npm: 4.2.8
Xcode: 8.2
Appcelerator Studio: 4.8.1.201612050850
```

**Pull request merged**